dbt_platform_helper/utils/versioning.py

```diff
@@ -137,24 +138,21 @@ def validate_template_version(app_version: Tuple[int, int, int], template_file_p
     )
 
 
-def validate_platform_helper_file_version(template_file_path: str):
-    validate_version_compatibility(
-        get_file_app_versions()[1],
-        get_template_generated_with_version(template_file_path),
-    )
-
-
 def generate_platform_helper_version_file(directory="."):
     base_path = Path(directory)
-    copilot_version = string_version(get_app_versions()[0])
-    click.echo(mkfile(base_path, ".platform-helper-version", f"{copilot_version}"))
+
+    package_version = string_version(get_platform_helper_versions()["package_version"])
+    click.echo(mkfile(base_path, PLATFORM_HELPER_VERSION_FILE, f"{package_version}"))
 
 
 def check_platform_helper_version_needs_update():
     if not running_as_installed_package() or "PLATFORM_TOOLS_SKIP_VERSION_CHECK" in os.environ:
         return
 
-    app_version, app_released_version = get_app_versions()
+    versions = get_platform_helper_versions()
+    app_version = versions["package_version"]
+    app_released_version = versions["platform_helper_file_version"]
```

Comment:
The var name doesn't seem to match the key? Later in this file you assign this value to `on_vile_version`. Should the key be `latest_pypi_release` or should var name change?

Reply:
Good point. I'll go through and rename to something more suitable. I think I left them as they were, but more descriptive names will definitely be better.

Reply:
Oh - yes, you were quite right. The key was wrong.